jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnectionFactory.java

```diff
@@ -34,18 +33,19 @@ public abstract class AbstractConnectionFactory extends ContainerLifeCycle imple
 {
     private final String _protocol;
     private final List<String> _protocols;
-    private int _inputbufferSize = 8192;
+    private int _inputBufferSize = 8192;
+    private boolean _useVirtualThreadToInvokeRootHandler;
```

Comment:
I think `_useVirtualThreads` is sufficient for the name.  Make it clear in the javadoc that we use virtual threads where appropriate to do so, specifically invoking root handler.